app/onboarding/plotstatus/page.tsx

```diff
@@ -0,0 +1,31 @@
+'use client';
+
+import { useProfile } from '@/hooks/useProfile';
+import {
+  Button,
+  ButtonContainer,
+  ContentContainer,
+  PageContainer,
+} from '../styles';
+
+export default function Page() {
+  const { addProfile } = useProfile();
+
+  const handleButtonClick = (state: string) => {
+    const newProfile = { state, email: '', phone_num: '', user_type: '' };
+    addProfile(newProfile);
+    console.log('test');
+  };
+  return (
+    <PageContainer>
+      <ContentContainer>
+        <h1>Plot Status</h1>
+        <h3>Do you own a plot to plant in?</h3>
+        <ButtonContainer>
+          <Button onClick={() => handleButtonClick('Yes')}>Georgia</Button>
```

Comment:
I think Yes and Georgia are flipped. Should be ('Georgia')}>Yes

Reply:
good point! I think it should be 
`<Button onClick={() => handleButtonClick('Yes')}>Yes</Button>` and the line below:
`<Button onClick={() => handleButtonClick('No')}>No</Button>`
Perhaps one way to reduce these types of errors is to create a list of possible options and map them into button components, so there's 1 source of truth for each button option